application/controllers/UserManagementController.php

```diff
@@ -378,8 +394,8 @@ public function actionUserPermissions()
             );
         }
 
-        $oRequest = Yii::app()->request;
-        $userId = $oRequest->getParam('userid');
+        $userId = Yii::app()->request->getParam('userid');
+        $userId = sanitize_int($userId);
```

Comment:
The same for that one. `$userId = sanitize_int(Yii::app()->request->getParam('userid');` Its just an idea but im not sure. Maybe you have another opinion about that.